### DISCUSSION TITLE
Snapdroid autoplay

### DISCUSSION THREAD

**Discussion by boris19** [snapcast/snapcast#1237] [Q&A]
HI!
is it possible to play music automatically after restarting the android device? without pressing the play button .

thank you for answers.

**Comment by badaix** [comment]
There is this "auto start snapclient" checkbox in settings that should do exactly this

**Reply by boris19** [reply]
Thank you for your answer. I was trying, but after restart, app started successfully and after i have physically push play button for stream music.

**Reply by badaix** [reply]
Sounds like a bug, which snapdroid version are you using?
